.travis.yml

```diff
@@ -9,6 +9,8 @@ matrix:
       env: NUMPY_VERSION="<1.14.0"
     - python: 3.6
     - python: 3.7
+      env: NUMPY_EXPERIMENTAL_ARRAY_FUNCTION="1"
+      env: NUMPY_VERSION="==1.16.2"
```

Comment:
I guess this is the issue. I don't have much experience with Travis, should the correct syntax be like below?

```
env:
  NUMPY_EXPERIMENTAL_ARRAY_FUNCTION="1"
  NUMPY_VERSION="==1.16.2"
```

Reply:
Something like this, I think:
```
env:
  - NUMPY_EXPERIMENTAL_ARRAY_FUNCTION="1"
  - NUMPY_VERSION="==1.16.2"
```

Reference: https://docs.travis-ci.com/user/environment-variables/

Reply:
Also add a new env for the `__array_function__` testing, leave the current python 3.7 one as-is.

Reply:
It's a bit confusing when to add the `-` or not. In Dask's `.travis.yml` there's also one before `env:`. I will try not to add one for now and follow exactly your suggestion.